app/src/components/search/LocationSearchBar.vue

```diff
@@ -100,6 +108,10 @@ export default Vue.extend({
         );
       } else return true;
     },
+    /**
+     * This event is triggert after every new input key. It will update the valie of the LocationSearchBar
+     * and if the whole input was deleted it will immediately start a search.
```

Comment:
The component does not know what happens if they change the value prop (thats what emit input do)